### DISCUSSION TITLE
Doesn't Boot Lineage Recovery Image

### DISCUSSION THREAD

**Discussion by BrettEBowman** [pgale/lineage_15.1_Installation_TSV#21] [Q&A]
I'm trying to flash a brand new ThinkSmart View with the stock ROM. I can get through all of the steps down to #20 where you are supposed to boot into the Recovery Mode. It always boots into the stock, Lenovo Recovery image, not the Lineage one. (Thanks for the picture of what the proper recovery mode screen should look like!)

The initial flashing seems to go just fine. No errors, etc. When I go into the Partition Manager tool, I don't get any errors. But, after selecting the file to load, it basically instantly returns. (No 3-5 seconds while loading.) So, I feel like that part might not be actually working properly.

Also, I notice when the Lenovo, stock recovery screen comes up, I do get an error at the bottom saying "E:Failed to save locale to /cache/recovery/last_locale: No such file or directory". Maybe that is just because it's the stock recovery, or maybe that indicates some other problem????

Does anyone have any ideas of what else I could try?

**Comment by pgale** [comment]
You could try the Lenovo recovery app first - on windows. I think I linked to it in the doc? Doesn't take too long and then try again.

**Comment by pgale** [comment]
And yes, it's definitely not installed the Lineage recovery image.

**Reply by BrettEBowman** [reply]
In your Linux instructions, it seems that in addition to writing the lineage-15.1-20240531-UNOFFICIAL-starfire-recovery.img file to the recovery partition, you also write emmc_appsboot.mbn to the aboot and abootbak partitions:
edl w aboot ~/ThinkSmartView/image/emmc_appsboot.mbn
edl w abootbak ~/ThinkSmartView/image/emmc_appsboot.mbn

Do I need to do those with Windows also?

**Comment by pimpinpanther96** [comment]
Try erasing the partition before writing. If it doesn't work, keep trying. Mine worked after several attempts
